docs/crio.8.md

```diff
@@ -177,7 +177,7 @@ If `hooks_dir` is unset, CRI-O will currently default to `/usr/share/containers/
 
 **--selinux**=**true**|**false**: Enable selinux support (default: false)
 
-**--seccomp-profile**="": Path to the seccomp json profile to be used as the runtime's default (default: "/etc/crio/seccomp.json")
+**--seccomp-profile**="": Path to the seccomp.json profile to be used as the runtime's default. If not specified, then the internal default seccomp profile will be used. (default: "")
```

Comment:
is the "(default: "") still necessary here?  We don't have it in crio.conf.5.md below.  I'd vote to remove it.

Reply:
Okay, I removed it as suggested. :+1: